python/baseline/tf/tfy.py

```diff
@@ -15,17 +15,31 @@ def _find_files_by_type(model_file, filetype):
     matching_files = []
 
     filetype_ending = "." + filetype
-    basepath = os.path.dirname(model_file)
-    if not os.path.isdir(basepath):
-        raise IOError("not a model directory")
+    basepath = get_basepath_or_cwd(model_file)
 
     for filename in os.listdir(basepath):
         if filename.endswith(filetype_ending):
             filename_without_ending = filename[:-len(filetype_ending)]
-            matching_files.append(filename_without_ending)
+            matching_files.append(os.path.join(basepath, filename_without_ending))
+
+    if not matching_files:
+        raise ValueError("no vocab files found in directory %s. \
+Please specify the model as path-like. e.g. /data/model/model-name-1234" % basepath)
 
     return matching_files
 
+def get_basepath_or_cwd(model_file):
```

Comment:
put to utils.py?

Reply:
if this belongs in utils.py, would _find_files_by_type also belong there?

Reply:
will merge for now, can figure out later if we should pull it up